charts/astronomer/templates/cli-install/cli-install-configmap.yaml

```diff
@@ -16,11 +16,53 @@ data:
     #! /usr/bin/env bash
 
     TAG=${1:-v{{ .Chart.Version }}}
-
+        
     if (( EUID != 0 )); then
         echo "Please run command as root."
         exit
     fi
 
     DOWNLOADER="https://raw.githubusercontent.com/astronomerio/astro-cli/master/godownloader.sh"
```

Comment:
@andriisoldatenko just want to make a note that we need to change this branch reference to `/main/` upon updating the CLI's default branch.